netlify/landings/documentation.yaml

```
@@ -0,0 +1,93 @@
+path: /documentation
+title: OpenAPI Documentation
+color: green
+relatedTags:
+  - blog-documentation
+hero:
+  bgColor: green
+  image:
+    alt: Documentation
+    src: /images/hubs/hero.png
+  subtitle: >-
+    Create beautiful, customizable, interactive API documentation generated from
+    your OpenAPI Specification
+  title: Hosted API Documentation
+featureSection:
+  actionBar:
+    ctas:
+      - color: accent
+        href: 'https://next.stoplight.io/'
+        title: Get Started Now
+    enabled: true
+    text: 'Ready to Publish? '
+  description: >-
+    Create API documentation in minutes. Combine OpenAPI Specifications
```

This has the same description as `/docs`. Sorry if I seem like a dog with a bone but why do we have /docs, /documentation and /hubs?
```suggestion
    Create API documentation in minutes. Combine OpenAPI descriptions
```